wifiphisher/common/interfaces.py

```diff
@@ -1,163 +1,147 @@
-#pylint: skip-file
 """
-This module was made to handle all the interface related operations for
-Wifiphisher.py
+This module was made to handle all the interface related operations of
+the program
 """
 
 import pyric
 import pyric.pyw as pyw
 
 
-class NotEnoughInterfacesFoundError(Exception):
-    """
-    Exception class to raise in case of a finding less than enough interfaces.
-    """
```

Comment:
Imagine this scenario where I have a wired interface and a wireless interface. It passes the two interface test while throwing the same error.